Jenkinsfile

```diff
@@ -955,3 +956,12 @@ def getVendorPatterns(beatName){
   }
   return output?.split('\n').collect{ item -> item as String }
 }
+
+def setGitConfig(){
+  sh(label: 'check git config', script: '''
+    if [ -z "$(git config --get user.email)" ]; then
+      git config user.email "58790750+beatsmachine@users.noreply.github.com"
```

Comment:
The `apm***` user got the same prefix, is `58790750+` correct? I don't know if it's a manual configuration done in GitHub or a coincidence that the email is the same one

Reply:
I've made a C&P, the email is not important, it is a no-reply email :)